iis-wf/iis-wf-export-actionmanager/src/test/java/eu/dnetlib/iis/wf/export/actionmanager/module/DocumentToSoftwareUrlActionBuilderModuleFactoryTest.java

```diff
@@ -48,43 +48,48 @@
 
     @Before
     public void initModule() {
-        DocumentToSoftwareUrlActionBuilderModuleFactory factory = new DocumentToSoftwareUrlActionBuilderModuleFactory();
-        module = (DocumentToSoftwareUrlActionBuilderModule) factory.instantiate(null, trustLevelThreshold, null);
+        factory = new DocumentToSoftwareUrlActionBuilderModuleFactory();
     }
 
     // ----------------------- TESTS --------------------------
 
     @Test(expected = NullPointerException.class)
-    public void test_build_null_object() throws Exception {
+    public void test_constructor_null_agent() throws Exception {
         // execute
-        module.build(null, agent, actionSetId);
+        factory.new DocumentToSoftwareUrlActionBuilderModule(trustLevelThreshold, null, actionSetId);
```

Comment:
This is unusual syntax. Just use `new DocumentToSoftwareUrlActionBuilderModule()`
This applies for every test for ActionBuilderModule.